content/blogposts/2024/announcing-prompt-library.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Announcing Prompt Library: save, reuse, and share frequently used prompts for Cody"
+authors:
+  - name: YK Sugi
+    url: https://x.com/ykdojo
+publishDate: 2024-11-25T10:00-01:00
+description: "Tired of typing the same prompts over and over again? Introducing Prompt Library. Save, reuse, and share prompts with your team."
+tags: [blog]
+slug: 'announcing-prompt-library'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/prompt-library/hero.webp
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/prompt-library/hero.webp
+---
+
+Through our customers' case studies and our conversations with them, we've discovered that engineering teams that adopt AI in their workflow tend to be about 30 to 40% more productive than the ones that do not. This may seem like a no-brainer, but changing a team's culture and letting them get over the mental hurdle of incorporating AI in their workflow is a significant challenge for many.
+
+In particular, we've heard from many customers and developers that their existing AI tool:
+- Doesn't always generate code that makes sense in their particular codebase.
+- Hallucinates because it doesn't use proper relevant context from your codebase and more.
+- Requires a steep learning curve for developers to be able to use it in a professional context.
+
+Today, we're introducing **Prompt Library** to tackle this problem. It will speed up AI adoption in your engineering team and it will encourage your team to share their knowledge about how to effectively use AI in your particular organization's workflow. That way, every single engineer in your team can get up to speed. Here's how it works:
```

Comment:
```suggestion
Today, we're introducing **Prompt Library** to tackle this problem. It will speed up AI adoption in your engineering organization and it will encourage your team to share their knowledge about how to effectively use AI across your specific workflows. That way, every single engineer on your team can quickly get up to speed and leverage AI in a uniform way. Here's how it works:
```